variables.tf

```
@@ -139,3 +113,9 @@ variable "extra_lb_http_tcp_listeners" {
   type        = list(any)
   default     = []
 }
+
+variable "tags" {
+  description = "Tags to apply to all resources created by the EKS Terraform module."
+  type        = any
```

```suggestion
  type        = map(string)
```

@Vampouille Another last minute request :) I did not notice this on my review yesterday.